eng/sendtohelix.proj

```diff
@@ -13,10 +13,12 @@
     <TimeoutInSeconds Condition="'$(TimeoutInSeconds)' == ''">600</TimeoutInSeconds>
     <_timeoutSpan>$([System.TimeSpan]::FromSeconds($(TimeoutInSeconds)))</_timeoutSpan>
     
-    <!-- We need to enable xunit reporter so that it parses test results -->
-    <EnableXunitReporter>true</EnableXunitReporter>
+    <!-- We need to enable xunit reporter so that it parses test results
+         Package testing doesn't run on xunit. -->
+    <EnableXunitReporter Condition="'$(IsPackageTesting)' != 'true'">true</EnableXunitReporter>
```

Comment:
Rather than making things differ on a property did you consider a common targets file with projects for the different purposes?

Reply:
That is a good idea. Since I want to get this in and outerloop runs the sooner, I will open an issue to cleanup sendtohelix.proj